pkg/operation/seed/components.go

```diff
@@ -175,15 +175,27 @@ func defaultIstio(ctx context.Context,
 	if err != nil {
 		return nil, err
 	}
+	gardenSeed := seed.GetInfo()
```

Comment:
Nit: Leave an empty line for readability

```suggestion

	gardenSeed := seed.GetInfo()
```